pamelalozano/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -161,7 +161,7 @@ private Comment getComment(HttpServletRequest request) {
     String msg = request.getParameter("msg");
     String author=null;
     if (userService.isUserLoggedIn()) {
-        author = userService.getCurrentUser().getEmail();
+        author = getUserNickname(userService.getCurrentUser().getUserId());
```

Comment:
What if a user wants to change their nickname later? Any old comments would still show their old nickname.

Reply:
Yeah, there are some things I have thought about but I don't have if there's time to implement, like nickname changes and repeated nicknames. I copied the whole nickname's part from the example in the walkthrough: 
https://github.com/pamelalozano16/STEP/tree/master/walkthroughs/week-4-libraries/authentication/examples/user-nicknames/src/main/java/com/google/sps/servlets

I'll try to figure this out by the end of the week if there's any time left, I think I need to start to implement another library.